src/ai/backend/agent/vendor/linux.py

```diff
@@ -30,6 +30,23 @@ def num_nodes():
         else:
             return 1
 
+    @staticmethod
+    def htcpu_info():
+        file_path = os.path.join("/sys/devices/system/cpu/cpu0/topology/thread_siblings_list")
```

Comment:
This is platform-dependent code. How about using a more platform-independent code by utilizing `psutil` like this reference: https://stackoverflow.com/a/62653110.

Reply:
Hmmm. But in that case, we should consider whether we expose the host's resources or the container's resources.  In the context of Backend.AI, we should use the host's resources and then we should detect the host-side CPU count and apply HT core exclusion logic.